mapping_project/lib/myhomepage.dart

```diff
@@ -30,10 +45,6 @@ class _MyHomePageState extends State<MyHomePage>
 
   List<String> suggestions = [
     "Saved",
```

Comment:
Is this the best way? Would it be better without an init value?

Reply:
This was for testing purposes, to show how search autocomplete shows when you type stuff in